test/components/tableTests.ts

```diff
@@ -294,77 +118,346 @@ describe("Tables", () => {
     let c5 = new Plottable.Component();
     let c6 = new Plottable.Component();
     let table: Plottable.Components.Table;
-    it("works in basic case", () => {
+
+    it("removes the specified Component", () => {
       table = new Plottable.Components.Table([[c1, c2], [c3, c4], [c5, c6]]);
       table.remove(c4);
-      assert.deepEqual((<any> table)._rows, [[c1, c2], [c3, null], [c5, c6]], "remove one element");
+      assertTableRows(table, [[c1, c2], [c3, null], [c5, c6]], "the requested element was removed");
+      assert.isNull(c4.parent(), "Component disconnected from the Table");
```

Comment:
I'm thinking of having a variable for the input array `[[c1, c2], [c3, c4], [c5, c6]]` and then have another variable that relates the input to the expected value `[[c1, c2], [c3, null], [c5, c6]]`